src/sentry/api/endpoints/project_transaction_threshold.py

```diff
@@ -5,12 +5,22 @@
 from sentry import features
 from sentry.api.bases.project import ProjectEndpoint, ProjectSettingPermission
 from sentry.api.serializers import serialize
-from sentry.models.transaction_threshold import TRANSACTION_METRICS, ProjectTransactionThreshold
+from sentry.models.transaction_threshold import (
+    TRANSACTION_METRICS,
+    ProjectTransactionThreshold,
+    TransactionMetric,
+)
+
+DEFAULT_THRESHOLD = {
```

Comment:
For now, just having it default to this- but we will want platform specific defaults soon